src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -168,7 +168,7 @@ private void CleanUpEndLocation(ITrackingSpan endTrackingSpan)
                 var line = SubjectBuffer.CurrentSnapshot.GetLineFromPosition(endSnapshotSpan.Start.Position);
                 var lineText = line.GetText();
 
-                if (lineText.Trim() == string.Empty)
+                if (string.IsNullOrWhiteSpace(lineText))
```

Comment:
This prevents the creation of a new string object that's done by `.Trim()`. And also will be faster.